file_paths.md

```diff
@@ -0,0 +1,108 @@
+Puppet Installation Layout
+==========================
+
+This table specifies the file paths in an installation of Puppet and the corresponding Puppet setting, denoted as `:setting`. The 3.x column shows the difference of the specification vs the actual 3.x implementation.
+
+puppet-agent
+------------
+
+    Path                                  Setting                        3.x
+    /etc/puppetlabs                       # :puppetdir                   n/a
```

Comment:
I vote that we not define a `puppetdir` setting. Less settings == more better, and people are free to configure `codedir` and `confdir` to suit their needs.